packages/core/contracts/common/implementation/AncillaryData.sol

```diff
@@ -29,6 +29,18 @@ library AncillaryData {
         return s;
     }
 
+    function toUtf8Bytes(bytes32 x) internal pure returns (bytes memory) {
```

Comment:
So this is the same implementation as `toUtf8BytesAddress` except that `s` is of size 64? Maybe we should refactor them to both use the same internal method `_createNewBytes(uint size)` to emphasize that the logic is the same

Reply:
also, some comments would be good! if you wrote this then awesome else we should reference where you got it from.

Reply:
@chrismaree agreed. I just copied from the other implementation in the file. Will add comments.

@nicholaspai it might be a bit harder to generalize because I had to change the size of the uint to cast to. Will think about this a little more tho.

Reply:
Done.